api/app/models/flag.rb

```diff
@@ -24,4 +24,13 @@ class Flag < ApplicationRecord
   # # Validations
   # validates :flaggable, presence: true
 
+  # Callbacks
+  after_create :notify_admin!
```

Comment:
Ensure that there is a spec for this behavior:
Flag enqueues an EnqueueFlaggedResourceJob job

Reply:
✔